src/ui/views/main/components/party/footer/index.tsx

```diff
@@ -45,18 +46,21 @@ export function PartyFooter() {
         <footer class="ln-party-footer">
             <div class="ln-party-footer__user">
                 <Show when={state.user} fallback={<Spinner size="100%" />}>
-                    {user => (<>
-                        <UserAvatar nickname={user.username} user={user} status={state.status} is_light_theme={state.is_light_theme} />
+                    {user => {
+                        let user_discriminator = user.discriminator.toString(16).toUpperCase().padStart(4, '0')
```

Comment:
The constraint of only running once also applies here, as this callback is effectively a nested component, and dependencies are untracked here.